db/schema.rb

```diff
@@ -250,6 +299,33 @@
     t.index ["updated_at"], name: "index_sessions_on_updated_at"
   end
 
+  create_table "statement_items", force: :cascade do |t|
+    t.bigint "statement_id", null: false
+    t.bigint "declaration_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "clawed_back_by_id"
+    t.string "state"
+    t.index ["clawed_back_by_id"], name: "index_statement_items_on_clawed_back_by_id"
+    t.index ["declaration_id"], name: "index_statement_items_on_declaration_id"
+    t.index ["statement_id"], name: "index_statement_items_on_statement_id"
+  end
+
+  create_table "statements", force: :cascade do |t|
+    t.integer "month"
+    t.integer "year"
+    t.date "deadline_date"
+    t.bigint "cohort_id", null: false
+    t.bigint "lead_provider_id", null: false
+    t.datetime "marked_as_paid_at"
+    t.decimal "reconcile_amount", precision: 8, scale: 2
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.datetime "frozen_at"
```

Comment:
@cwrw to confirm the frozen concept

Reply:
so I can confirm now we are using `type` to freeze a statement, so when it becomes payable it is "frozen". We are not using type here, we can either rely on deadline date OR this field, so happy to keep for now